### ISSUE TITLE
`fields` query parameter does not work consistently across all response formats

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/registry-api#172]
## 🐛 Describe the bug

```
curl --get https://pds.nasa.gov/api/search-sbnumd/1.0/bundles/urn:nasa:pds:gbo-kpno::4.0/collections --data-urlencode 'fields=Time_Coordinates.start_date_time' | json_pp
```
returns all fields

```
curl --get 'https://pds.nasa.gov/api/search-sbnumd/1.0/bundles/urn:nasa:pds:gbo-kpno::4.0/collections' --data-urlencode 'fields=pds:Time_Coordinates.pds:start_date_time' --header "Accept: text/csv"
```
returns a bunch of whitespace

## 🕵️ Expected behavior

This must be consistent 100% of the time. Even if we just support via text/csv and kvp+json, that will suffice for now. But that doesn't currently work.

## 📚 Version of Software Used

1.0.2

## 🖥 System Info

---

## 🦄 Related requirements
related to https://github.com/NASA-PDS/pds-api/issues/209

**Comment by al-niessner** [comment]
fixed on branch 1.0.3

**Comment by jordanpadams** [comment]
fixed per https://github.com/NASA-PDS/registry-api/pull/175 (I think?). let's test it out and see...
